src/libertem/common/numba.py

```diff
@@ -248,27 +248,31 @@ def enable_caching(self):
 def prime_numba_cache(ds):
     dtypes = (np.float32, None)
     for dtype in dtypes:
-        roi = np.zeros(ds.shape.nav, dtype=bool).reshape((-1,))
-        roi[0] = 1
+        roi = np.ones(ds.shape.nav, dtype=bool).reshape((-1,))
```

Comment:
Would `roi = None` work? It should give the same result as `roi = np.ones(...)`, but skip the whole ROI-related logic of datasets.